rfcs/README.md

```diff
@@ -0,0 +1,107 @@
+
+## Before creating an RFC
+
+[Before creating an RFC]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality proposals, proposals for previously-rejected features, or those that don't fit into the near-term roadmap, may be quickly rejected, which can be demotivating for the unprepared contributor. Laying some groundwork ahead of the RFC can make the process smoother. 
+
+Although there is no single way to prepare for submitting an RFC, it is generally a good idea to pursue feedback from other project developers beforehand, to ascertain that the RFC may be desirable; having a consistent impact on the project requires concerted effort toward consensus-building. 
+
+The most common preparations for writing and submitting an RFC includes talking the idea over on our [community Discord server], or in relevant Issue threads when the idea stems from previous discussion.
+
+As a rule of thumb, receiving encouraging feedback from long-standing project developers, and particularly members of the core team is a good indication that the RFC is worth pursuing.
+
+## What the process is
+
+[What the process is]: #what-the-process-is
+
+In short, to get a major feature added to the Block Protocol, one must first get the RFC merged into the [`/rfcs`](../rfcs) folder as a markdown file. At that point the RFC is "active" and may be implemented with the goal of eventual inclusion into the Block Protocol.
+
+- Fork the Block Protocol repo https://github.com/blockprotocol/blockprotocol
+- Copy [`rfcs/0000-template.md`](0000-template.md) to `rfcs/text/0000-my-feature.md` (where
+'my-feature' is descriptive. Don't assign an RFC number yet).
+- Fill in the RFC. Put care into the details: **RFCs that do not present convincing motivation, demonstrate understanding of the impact of the design, or are disingenuous about the drawbacks or alternatives tend to be poorly-received**.
+- Submit a pull request. As a pull request the RFC will receive design feedback from the larger community, and the author should be prepared to revise it in response.
+- Now that your RFC has an open pull request, use the issue number of the PR to update your `0000-` prefix to that number
+- Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
+- The core team will discuss the RFC pull request, as much as possible in the comment thread of the pull request itself. Offline discussion will be summarized on the pull request comment thread.
+- RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown. You can make edits, big and small, to the RFC to clarify or change the design, but make changes as new commits to the pull request, and leave a comment on the pull request explaining your changes. Specifically, **do not squash or rebase commits after they are visible on the pull request**.
```

Comment:
we should have a branch format that _we_ use for RFCs, because then we can protect the ones we create according to the pattern and prevent ourselves accidentally force pushing

Reply:
Lets go with `{initial}/rfc-{branch_name} so e.g. "am/rfc-my-feature"